tests/datasource/data_connector/test_data_connector_query.py

```diff
@@ -327,7 +326,7 @@ def my_custom_batch_selector(batch_identifiers: dict) -> bool:
             data_connector_name="general_filesystem_data_connector",
             data_asset_name="TestFiles",
             batch_identifiers=IDDict(
-                {"name": "james", "timestamp": "20200810", "price": "1003"}
+                {"name": "james", "timestamp": "20200811", "price": "1009"}
             ),
```

Comment:
we had configured `DateTimeSorter` as `descending` which means `20200811` should have been part of the output, rather than `20200810`.

Reply:
Just to confirm, this was previously erroneous, which is why it was  `20200810` and now that you've fixed it, we have the appropriate output of `20200811`?

Reply:
yes that's correct .. it would have been `name` sorted ascending Lexicographically, then `timestamp` sorted descending DateTime (which means `20200811` should have come before `20200810`)